calm_adapter/calm_api_client/src/test/scala/uk/ac/wellcome/platform/calm_api_client/CalmApiClientTest.scala

```diff
@@ -131,17 +128,17 @@ class CalmApiClientTest
 
   it("fails on error responses from the API") {
     val responses = List(HttpResponse(500, Nil, "Oops", protocol))
-    withApiClient(responses) {
+    withCalmClients(responses) {
       case (apiClient, _) =>
         whenReady(apiClient.search(query).failed) { error =>
-          error.getMessage shouldBe "Unexpected status from CALM API: 500 Internal Server Error"
+          error.getMessage shouldBe "Max retries attempted when calling Calm API"
```

Comment:
I changed the test fixtures to all use a client with exponential backoff (as is used IRL) rather than using a one-shot client fixture. I may come back round later and see if I can clean things up having removed that one-shot case.